packages/service-library/tests/conftest.py

```diff
@@ -61,3 +65,26 @@ def fake_data_dict(faker: Faker) -> dict[str, Any]:
     }
     data["object"] = deepcopy(data)
     return data
+
+
+@pytest.fixture
+async def get_redis_client_sdk(
```

Comment:
1. why a factory fixture?
2. use `create_` instead of `get` since you are constructing and init a client